docker-compose.yml

```diff
@@ -21,15 +21,13 @@ services:
       - "3001:3000"
     volumes:
       - bundle:/vendor/bundle
-      - ./:/home/app
-      - ../meilisearch-ruby:/home/meilisearch-ruby
+      - ./:/home/package
+      # - ../meilisearch-ruby:/home/meilisearch-ruby
 
   meilisearch:
-    image: getmeili/meilisearch:v0.26.1
-    tty: true
-    stdin_open: true
+    image: getmeili/meilisearch:latest
```

Comment:
Do we want latest or a fixed version until we officially release a version of the package compatible with the latest version of Meilisearch?

Reply:
I think we can keep it the latest, because, during the pre-release time, we can just change to the correct version. We already do that today, correct?